tools/clang/test/DXC/scalarrepl-param-hlsl-const-to-local-and-back.hlsl

```diff
@@ -0,0 +1,46 @@
+// RUN: %dxc -T ps_6_2 %s | FileCheck %s
```

Comment:
```suggestion
// RUN: not %dxc -T ps_6_2 %s 2>&1 | FileCheck %s
```
LIT `RUN` lines are basically just shell commands with some extra variable expansions. If the sell command returns success the test passes if it returns failure it fails.

Prefacing the call with `not` inverts the return code of the process (success becomes failure, failure becomes success). This allows LIT to run commands that are expected to fail and report an error if they succeed.

Adding `2>&1` is fun POSIX shell for redirect standard error to standard out, which we then pipe into FileCheck, so that below we can check the output.

Reply:
Done.